packages/main/src/resolvers/account.test.ts

```diff
@@ -319,6 +319,59 @@ describe("sync", () => {
       expect.any(ConnectionManager)
     )
   })
+  it("deletes an account", async () => {
```

Comment:
Sorry to ask for such a trivial change, but I would like to have blank lines between tests.